### DISCUSSION TITLE
Statistics are not reset to zero?

### DISCUSSION THREAD

**Discussion by fizsef** [AdguardTeam/AdGuardHome#6424] [Q&A]
In the settings, it is selected to save statistics for 24 hours, but the calculation never starts from scratch. At what time does the countdown of these 24 hours begin, why is the statistics counter reset to zero?

**Comment by ainar-g** [comment]
I'm not sure what you mean by “starting from scratch”.  The statistics interval defines the duration of the sliding window for the statistics period.  It isn't reset to zero after 24 hours, but rather retains only the information for the last 24 hours, deleting statistics beyond that.

**Comment by fizsef** [comment]
A more correct option would be when the statistics will be reset to zero and start again after the time specified in the settings has expired. Otherwise, it is not very clear, for example, statistics show 100 requests, I send 10 more, but he began to show 90, although I logically expect to see 110.

**Comment by fizsef** [comment]
What command can I use to reset statistics in OpenWRT ? I want to register such a command in Cron in order to reset statistics at a certain time.

**Reply by ainar-g** [reply]
You can either use the HTTP API or stop AGH, remove `data/stats.db`, and restart it.
